apps/storybook/stories/design/Icon.mdx

```diff
@@ -0,0 +1,27 @@
+import { Meta, Canvas, Controls } from "@storybook/addon-docs";
+import * as IconStories from "./Icon.stories";
+
+<Meta of={IconStories} />
+
+# Icon
+
+Vi benytter ikoner som et virkemiddel for hjelpe brukere med å forstå handling eller kontekst. Ikoner kan være med på å skille og fremheve innhold.
+Det er også et fint navigasjonsverkøy som kan vise varsling av feil eller fortelle brukeren om de forlater nettsiden.
```

Comment:
Her bør vi også ha med en lenke til noe som forteller om riktig bruk av ikoner. Her er et eksempel: https://a11y-101.com/design/icons

Det er mulig at Wanni har en enda bedre side, hun har nevnt noe om at Material har en egen guideline for dette.